res/fingerprint-server-api.yaml

```diff
@@ -299,6 +299,7 @@ paths:
             issued by Fingerprint Pro.
         - name: request_id
           in: query
+          x-go-skip-pointer: true
```

Comment:
Could you explain what `x-go-skip-pointer` parameter does? It apparently works like [x-go-type-skip-optional-pointer](https://github.com/oapi-codegen/oapi-codegen?tab=readme-ov-file#openapi-extensions) but I can't find anything about it on the web. 

Also, could you explain why we are adding them to **Get visits by visitor ID** API? The PR says it fixes nullable values for `FingerprintApiSearchEventsOpts` but it changes another API.

Reply:
It's a custom flag, so you won't find anything about it on the web 😁
My koala PR describes pretty well what this flag does: https://github.com/fingerprintjs/koala/pull/9122